src/components/Table/Table.test.tsx

```diff
@@ -215,6 +216,42 @@ describe('Table Component', () => {
     expect(onOverview).toHaveBeenCalledWith(props.data[3], 3, expect.any(Object))
   })
 
+  test('renders edit and delete action correctly when passing onClick in action', async() => {
+    const onEditRow = jest.fn()
+    const onDeleteRow = jest.fn()
+
+    render(
+      <Table
+        {...props}
+        actions={[{
+          dataIndex: Action.Edit,
+          onClick: onEditRow,
+        }, {
+          dataIndex: Action.Delete,
+          onClick: onDeleteRow,
+        }]}
+      />
+    )
+
+    const editButtons = screen.getAllByRole('button', { name: 'PiPencilSimpleLine' })
+    const deleteButtons = screen.getAllByRole('button', { name: 'PiTrash' })
+
+    expect(editButtons).toHaveLength(props.data.length)
+    expect(deleteButtons).toHaveLength(props.data.length)
+
+    const [editButton] = editButtons
+    expect(editButton).toBeVisible()
+    fireEvent.click(editButton)
```

Comment:
can we use `userEvent`?

Reply:
I've used fireEvent to be coherent with the other tests in the file